web/src/context/store.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { createContext, useReducer } from 'react';
+import { addToCart, clearFromCart, emptyCart, getCart, removeOneFromCart } from '../lamda-services/use-local-storage';
```

Comment:
can you move them from lamda-services? this folder should be strictly used for backend api things, this is only frontend